wrappers/python/nntile/model/llama.py

```diff
@@ -56,7 +56,7 @@ def __init__(self,
 
         for dec_layer in decoders:
             activations.extend(dec_layer.activations[1:])
-            layers.extend(dec_layer.layers)
+            layers.extend([dec_layer])
```

Comment:
layers cannot contain objects of class `BaseModel` as of now

Reply:
I believe we need to allow it to contain BaseModel
Otherwise, it is very unclear, where all submodels are

Reply:
ok, discussed above. Removed this